core/src/main/java/brooklyn/event/feed/AttributePollHandler.java

```diff
@@ -52,6 +54,7 @@
     @SuppressWarnings("rawtypes")
     private final AttributeSensor sensor;
     private final AbstractFeed feed;
+    private final boolean suppressDuplicates;
```

Comment:
What are the compelling reasons to ever *include* duplicates? Should they be suppressed by default, with a config option to explicitly enable them?